src/module.c

```diff
@@ -531,6 +531,69 @@ Module *Module::parse()
             error("has non-identifier characters in filename, use module declaration instead");
     }
 
+    // Add internal used functions in 'object' module members.
+    if (!parent && ident == Id::object)
+    {
+        static const utf8_t code_ArrayEq[] =
+            "bool _ArrayEq(T1, T2)(T1[] a, T2[] b) {\n"
+            " if (a.length != b.length) return false;\n"
+            " foreach (size_t i; 0 .. a.length) { if (a[i] != b[i]) return false; }\n"
+            " return true; }\n";
+
+        static const utf8_t code_ArrayPostblit[] =
+            "void _ArrayPostblit(T)(T[] a) { foreach (ref T e; a) e.__xpostblit(); }\n";
+
+        static const utf8_t code_ArrayDtor[] =
+            "void _ArrayDtor(T)(T[] a) { foreach_reverse (ref T e; a) e.__xdtor(); }\n";
+
+        static const utf8_t code_xopEquals[] =
+            "bool _xopEquals(in void*, in void*) { throw new Error(\"TypeInfo.equals is not implemented\"); }\n";
+
+        static const utf8_t code_xopCmp[] =
+            "bool _xopCmp(in void*, in void*) { throw new Error(\"TypeInfo.compare is not implemented\"); }\n";
+
+        Identifier *arreq = Id::_ArrayEq;
+        Identifier *xopeq = Identifier::idPool("_xopEquals");
+        Identifier *xopcmp = Identifier::idPool("_xopCmp");
+        for (size_t i = 0; i < members->dim; i++)
+        {
+            Dsymbol *sx = (*members)[i];
+            if (!sx) continue;
+            if (arreq && sx->ident == arreq) arreq = NULL;
+            if (xopeq && sx->ident == xopeq) xopeq = NULL;
+            if (xopcmp && sx->ident == xopcmp) xopcmp = NULL;
+        }
+
+        if (arreq)
+        {
+            Parser p(loc, this, code_ArrayEq, strlen((const char *)code_ArrayEq), 0);
+            p.nextToken();
+            members->append(p.parseDeclDefs(0));
+        }
+        {
+            Parser p(loc, this, code_ArrayPostblit, strlen((const char *)code_ArrayPostblit), 0);
+            p.nextToken();
+            members->append(p.parseDeclDefs(0));
+        }
+        {
+            Parser p(loc, this, code_ArrayDtor, strlen((const char *)code_ArrayDtor), 0);
+            p.nextToken();
+            members->append(p.parseDeclDefs(0));
+        }
+        if (xopeq)
+        {
+            Parser p(loc, this, code_xopEquals, strlen((const char *)code_xopEquals), 0);
+            p.nextToken();
+            members->append(p.parseDeclDefs(0));
+        }
+        if (xopcmp)
+        {
+            Parser p(loc, this, code_xopCmp, strlen((const char *)code_xopCmp), 0);
+            p.nextToken();
+            members->append(p.parseDeclDefs(0));
+        }
+    }
```

Comment:
It would be better, but this is the most easy way that I can implement quickly. This change is for the regression issue, so I should not use risky way.